### PR TITLE
[ATMOSPHERE-361] doc: Add versions to README

### PR DESCRIPTION
**Pull request by gtirloni**

### CODE REVIEW

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ name: image
 on:
   pull_request:
     paths-ignore:
+      - README.md
+      - doc/**
```
Comment (mnaser) [inline .github/workflows/image.yml:7]: We're moving away from GitHub actions so let's drop this.

File: zuul.d/container-images/base.yaml [hunk 2]
```diff
@@ -24,7 +24,9 @@
     name: atmosphere-buildset-registry
     parent: ci-buildset-registry
     irrelevant-files:
+      - ^README\.md$
       - ^doc/
+      - \.github/
```
Comment (mnaser) [inline zuul.d/container-images/base.yaml:29]: Let's drop this.

File: zuul.d/container-images/base.yaml [hunk 3]
```diff
@@ -46,7 +48,9 @@
     parent: ci-upload-container-image
     abstract: true
     irrelevant-files:
+      - ^README\.md$
       - ^doc/
+      - \.github/
```
Comment (mnaser) [inline zuul.d/container-images/base.yaml:53]: ..and this..

File: zuul.d/container-images/base.yaml [hunk 4]
```diff
@@ -57,7 +61,9 @@
     name: atmosphere-promote-container-image
     parent: ci-promote-container-image
     irrelevant-files:
+      - ^README\.md$
       - ^doc/
+      - \.github/
```
Comment (mnaser) [inline zuul.d/container-images/base.yaml:66]: and this :)

File: zuul.d/jobs.yaml [hunk 5]
```diff
@@ -12,7 +12,9 @@
     parent: build-ansible-collection
     pre-run: zuul.d/playbooks/build-collection/pre.yml
     irrelevant-files:
+      - ^README\.md$
       - ^doc/
+      - \.github/
```
Comment (mnaser) [inline zuul.d/jobs.yaml:17]: this too

File: zuul.d/jobs.yaml [hunk 6]
```diff
@@ -27,7 +29,9 @@
     run: zuul.d/playbooks/molecule/run.yml
     post-run: zuul.d/playbooks/molecule/post.yml
     irrelevant-files:
+      - ^README\.md$
       - ^doc/
+      - \.github/
```
Comment (mnaser) [inline zuul.d/jobs.yaml:34]: this to

**Review by mnaser — CHANGES_REQUESTED** [review]

**Comment by gtirloni** [comment]
@mnaser ready for review

**Pull request closed without merging**